DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -96,17 +96,23 @@
 DQMOfflinePhysics = cms.Sequence( dqmPhysics )
 
 
+from Validation.Geometry.trackingRecoMaterialAnalyzer_cfi import materialDumperAnalyzer
```

Comment:
ciao @davidlange6 ,
you maybe noticed already that there are already other 2 dependencies on validation, besides this new one.
In any case, I'll think about the best approach: either move code in DQM or config in Validation and update this PR.